spec/factories/users.rb

```diff
@@ -0,0 +1,17 @@
+def lowercase_alphabets
+  ('a'..'z').to_a
+end
+
+FactoryBot.define do
+  factory :user do
+    sequence :email do |n|
+      "tester#{lowercase_alphabets[n - 1]}@example.com"
```

Comment:
これだと最大で 26 データしか作れなくない？
↓数値ではダメ？

```suggestion
      "tester-#{n}@example.com"
```

Reply:
ユーザー名はアルファベットで構成する制約があるので、数値は入れられません。